documentation/articles/contributing.md

```diff
@@ -0,0 +1,139 @@
+
+#### The First Sentence Is the Most Important
+
+There is no guarantee that the reader will read beyond the first sentence of the reference documentation.  This is why that first sentence is the most important and should convey the most key piece of information.  Take your time to write the most concise and clear first sentence possible.  This helps users tremendously and goes a long way towards having great documentation.
+
+#### Surface Information Hidden in the Code
+
+Being inline with the code allows you to easily look for critical information within it that the user might not know from looking at the API alone.  Take your time to explore inner method calls and platform specific sections of the code.  The time to write the documentation is once you feel you fully understand the code you are documenting.  If you don't feel you understand the code then leave the documentation for someone else to write.
+
+#### Focus on what adds value to the consumer
+
+Limit documentation to public methods and functions unless there is a specific reason to include internal methods, while documenting internals helps with readability of the code, it provides limited use to consumers of the MonoGame Framework.
+
+#### Documentation Is Referenced Not Read
+
+Remember that the user is searching for an answer for a specific question.  It is your job to predict these questions and provide them clear answers.
+
+#### XML Tag guidance
+
+By default, the standard [Microsoft recommendations](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags) should be used for filling in XML tags for each class, method and property.
+
+With a few points to call out:
+
+#### See / CRef should be used whenever an API reference is used in the documentation
+
+To ensure that API documentaiton is linked to whichever reference is used, `<see>` and `<cref` references should be used, this helps users navigate the methods, espeically when looking up initializers or use of a property or method.
+
+#### 120 Width comments for easy reading
+
+Comments should be limited to **120** width, with overflow moving to the next line to make reading easier, for example:
+
+```csharp
+<summary>
+Packed vector type containing unsigned normalized values ranging from 0 to 1. The x and z components use 5 bits,
+and the y component uses 6 bits.
+</summary>
+```
+
+#### Use the packed multi-line stype with surrounding tags
+
+To keep the documentation packed and readable, each parameter should be contained to a single line, for example:
+
+```csharp
+<summary>
+Creates a new instance of Bgr565.
+</summary>
+<param name="x">The x component</param>
+<param name="y">The y component</param>
+<param name="z">The z component</param>
```

Comment:
Should a note or exception be added here when a parameter description exceeds the 120 character width?

For example. the `ContentManager` documentation would currently look like the following

```csharp
        /// <summary>
        /// 
        /// </summary>
        /// <param name="serviceProvider">The service provider that the <see cref="ContentManager"/> should use to locate services.</param>
```

With the guidelines on using the `<see cref>` tag here, this causes the `<pram>` line to extend a total of 140 characters (including the indention due to the namespace and class level indentions

(image attached shows vertical lines at the 80 and 120 character width limits)
![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/08e81848-15c4-426a-8a78-23d85b4e5b43)

In these situations would it be ok to split that parameter to a multi line such as 

```csharp
        /// <summary>
        /// 
        /// </summary>
        /// <param name="serviceProvider">
        /// The service provider that the <see cref="ContentManager"/> should use to locate services.
        /// </param>
```

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/dc15d74a-4f18-4606-b257-8777a03899ad)